src/scss/06_components/paragraphs/_hero.scss

```diff
@@ -49,6 +49,10 @@ $koro-height: 50px;
   }
 }
 
+.hero__publish-information {
```

Comment:
There's an empty space before the ", updated" text:
![image](https://github.com/user-attachments/assets/528490ae-f78c-476a-a90c-7d2d8a50ec47)

Reply:
Fixed the unwanted spacing before the datetime on the template.